inc/slalevel_ticket.class.php

```diff
@@ -214,7 +214,8 @@ static function doLevelForTicket(array $data, $sltType) {
                   // Put next level in todo list
                   $next = $slalevel->getNextSltLevel($ticket->fields[$sltField],
                                                      $data['slalevels_id']);
-                  $slt->addLevelToDo($ticket, $next);
+                  $current = $data['slalevels_id'];
+                  $slt->addLevelToDo($ticket, $next, $current);
```

Comment:
Do not know if this solve any issue; but `addLevelToDo` takes 2 arguments; 3 are passed here.